test/tools/unified-spec-runner/operations.ts

```diff
@@ -173,7 +173,10 @@ operations.set('assertNumberConnectionsCheckedOut', async ({ entities, operation
 
 operations.set('bulkWrite', async ({ entities, operation }) => {
   const collection = entities.getEntity('collection', operation.object);
-  return collection.bulkWrite(operation.arguments.requests);
+  const operations = operation.arguments.requests;
```

Comment:
You can do this a little bit more concisely like in some other operations here by doing:
`const { requests: operations, ...options } = operation.arguments;`